### PR TITLE
WSG-94 refactor questionnaire generator. add questionnaire command

### PR DESCRIPTION
**Pull request by xkmato**
- Adds questionnaire cli command
- Refactors questionnaire generator class

## To Test

 who_l3_smart_tools/cli/questionnaire.py -i tests/data/l2/test_dd.xlsx

### CODE REVIEW

File: who_l3_smart_tools/utils/jinja2.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    "Quantity": "integer",
+}
+
+
+def initalize_jinja_env(file_name):
+    template_dir = os.path.join(
+        os.path.dirname(os.path.abspath(file_name)), "templates"
+    )
+    return Environment(
+        loader=FileSystemLoader(template_dir),
```
Comment (ibacher) [inline who_l3_smart_tools/utils/jinja2.py:21]: Have you considered using the PackageLoader for this? I found it pretty simple to use, e.g.,

```python
def initalize_jinja_env(module_name):
    return Environment(
        loader=PackageLoader(module_name), # defaults to using "templates" folder inside the package
        trim_blocks=True,
        lstrip_blocks=True,
    )
```

Then you replace:

```python
jinja2_env = initalize_jinja_env(__file__)
```

With:

```python
jinja2_env = initalize_jinja_env(__name__)
```

File: who_l3_smart_tools/core/questionnaires/templates/questionnaire.j2
Comment (ibacher) [inline who_l3_smart_tools/core/questionnaires/templates/questionnaire.j2:1]: I kind of prefer the naming convention `<template_name>.<ext>.j2` so `questionnaire.fsh.j2`

**Review by ibacher — APPROVED** [review]
Couple of small things, but basically LGTM.

File: who_l3_smart_tools/core/questionnaires/templates/questionnaire.j2
Comment (xkmato) [reply who_l3_smart_tools/core/questionnaires/templates/questionnaire.j2:1]: I agree. Thanks

**Pull request merged**